### DISCUSSION TITLE
Does the latest engine (WS11WineCX64Bit23.7.1) include the CX version of MoltenVK?

### DISCUSSION THREAD

**Discussion by sofakng** [The-Wineskin-Project/WineskinServer#527] [Q&A] [answered]
Does the latest engine (WS11WineCX64Bit23.7.1) include the CX version of MoltenVK?

I remember reading that VKD3D required the custom version of MoltenVK from CX.

Also, does this version of MoltenVK have a version number?

**Comment by Gcenx** [comment]
No Engines usually won’t include dylibs unless there absent from the current master wrapper.

**Reply by Gcenx** [reply]
Alao something to note is that VKD3D is only able to play Diablo2:Resurrected & Diablo4 currently.

**Reply by sofakng** [reply]
Oh, sorry... yes I'm aware it only works for Diablo 2 and Diablo 4.

The wrapper includes MoltenVK then?  It looks like the latest version is 2.9.2.0.  Does this include the latest CX custom version of MoltenVK?

**Reply by Gcenx** [reply, marked as answer]
Try the attached that should be the equivalent to the one shipped in CX23.7.1 [libMoltenVK.dylib.zip](https://github.com/Gcenx/WineskinServer/files/13933337/libMoltenVK.dylib.zip)

Place that into /SharedSupport/wine/lib as the current build of CX23.7.1 for some reason it doesn't like it place into /Frameworks, currently investigating that.

**Reply by sofakng** [reply]
Thanks so much!  Does your CX23.7.1 engine already include the latest VKD3D as well?

**Reply by Gcenx** [reply]
> Thanks so much! Does your CX23.7.1 engine already include the latest VKD3D as well?

I don’t think VKD3D changed from CX23.0.0 > 23.7.1 only MoltenVK so it _should_ already match CrossOver-23.7.1 when the same copy of libMoltenVK is used

**Reply by sofakng** [reply]
Thanks so much for the help and information!
